progress.go

```diff
@@ -11,16 +11,25 @@ import (
 	"sync/atomic"
 )
 
-const defaultChunkDir = ".chunk"
+// DefaultChunkDir is the directory where Chunk keeps track of each chunk
+// downloaded of each file. It us created under the user's home directory by
+// default. It can be replaced by the environment variable CHUNK_DIR.
+const DefaultChunkDir = ".chunk"
 
 // get the chunk directory under user's home directory
 // TODO: make it configurable (maybe an envvar?)
-func getChunkDirectory() (string, error) {
-	u, err := user.Current()
-	if err != nil {
-		return "", fmt.Errorf("could not get current user: %w", err)
+func getChunkDirectory(custom string) (string, error) {
+	d := os.Getenv("CHUNK_DIR")
```

Comment:
Please move this to after check `custom` (i.e., line 26).

Reply:
Got it 11c608a